eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -47,13 +47,13 @@ library
         bytestring,
         cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.5.1,
         cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.10,
+        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.5 && <1.11,
         cardano-ledger-pretty,
-        cardano-ledger-allegra ^>=1.2,
-        cardano-ledger-shelley:{cardano-ledger-shelley, testlib} >=1.6 && <1.9,
+        cardano-ledger-allegra >=1.2.5.1,
```

Rule of thumb: patch version should never appear in bounds, because pathc changes are not visible to the user should never affect the decision on choosing a version.

Also, there is nothing new in this package that needs anything from `cardano-ledger-allegra`
```suggestion
        cardano-ledger-allegra >=1.2,
```